tests/infrastructure/instance_types/test_common_vm_preference.py

```diff
@@ -103,15 +105,20 @@ def run_general_vm_preferences(client, namespace, preferences):
             )
 
 
+@pytest.fixture(scope="module")
+def vm_preferences_list():
+    file_name = "tests/infrastructure/instance_types/vm_preference_list.json"
+    try:
+        with open(file_name, "r") as file:
+            data = json.load(file)
+        return data
+    except (FileNotFoundError, json.JSONDecodeError):
+        pytest.fail(f"Could not read expected preferences names from file: {file_name}")
```

Comment:
Reading a file is an expensive operation, why do you need a file? You can store everything in a python dictionary.

Reply:
You are right, I will move it to a .py file and insert it under a constant, we still want to separate it into a different file making it easier to manage.